mobile/src/src/components/Housing/Listings/ListingsContainer.jsx

```diff
@@ -42,18 +42,53 @@ class ListingContainer extends Component {
   };
 
   likeListing = (id, shouldLike) => {
+    // updates number of likes
+    let index = this.state.listings.findIndex((obj) => obj.id == id);
+    let copyList = JSON.parse(JSON.stringify(this.state.listings));
+    shouldLike == true
+      ? (copyList[index].likes = this.state.listings[index].likes + 1)
+      : (copyList[index].likes = this.state.listings[index].likes - 1);
+    this.setState({ listings: copyList });
+
     ListingsEndpoint.like(
       id,
       shouldLike,
       () => {},
```

Comment:
If you haven't already, maybe consider updating the number of likes in the state object in this handleSuccess method. Was this a deliberate choice in order to make the displayed number of likes update immediately instead of having to wait for the Axios request?